core/camel-util/src/main/java/org/apache/camel/util/SkipLastByteInputStream.java

```diff
@@ -0,0 +1,92 @@
+package org.apache.camel.util;
```

Comment:
Add license header

Reply:
Thanks, fixed in 44744b2ad8d83f2e110bd7c789cc504c6b6676ab